### PR TITLE
Add test case for bug fix of message panel tag #459

### PR DESCRIPTION
**Pull request by btkukinom**
Please review #459 .

In the case of empty in paginationTag, it was thought not to be output to the JSP.

### CODE REVIEW

File: terasoluna-gfw-functionaltest-web/src/main/resources/i18n/application-messages_ja.properties [hunk 1]
```diff
@@ -5,6 +5,7 @@ i.gt.me.0003 = \u6210\u529F\uFF01\uFF01
 i.gt.me.0004 = \u60C5\u5831\uFF01\uFF01
 i.gt.me.0005 = \u3053\u3093\u306B\u3061\u306F\u3000{0}\uFF01\uFF01
 i.gt.me.0006 = <div>\u3053\u3093\u306B\u3061\u306F\u3000\u4E16\u754C\uFF01\uFF01</div>
+i.gt.me.0007 =
```
Comment (iwanagah) [inline terasoluna-gfw-functionaltest-web/src/main/resources/i18n/application-messages_ja.properties]: - [ ] メッセージ空でいいんでしたっけ？

**Comment by iwanagah** [comment]
@btkukinom Please check my comment.

@btkatoutmj Please review this pull request.

File: terasoluna-gfw-functionaltest-web/src/main/resources/i18n/application-messages_ja.properties (hunk 1 above)
Comment (btkukinom) [inline terasoluna-gfw-functionaltest-web/src/main/resources/i18n/application-messages_ja.properties]: I think this.
If I set the empty string in the message, "<li> </ li>" is to make sure that the output.

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/message/MessageTest.java [hunk 2]
```diff
@@ -732,4 +732,41 @@ public void test09_02_disableHtmlEscape() {
...
+    public void test10_01_tagConfiguration() {
+        driver.findElement(By.id("tagConfiguration_10_01")).click();
+
+        // innerElement close with </li> confirm
+        assertThat(driver.getPageSource().contains(
+                "<div class=\"alert alert-success\"><ul><li></li></ul></div>"),
+                is(true));
+
+        // screen capture
+        screenCapture.save(driver);
```
Comment (btkatoutmj) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/message/MessageTest.java]: I think, these three cases are no difference in the appearance of the result screen.
So, it should take page source for evidence.

File: terasoluna-gfw-functionaltest-web/src/main/resources/i18n/application-messages_ja.properties (hunk 1 above)
Comment (iwanagah) [inline terasoluna-gfw-functionaltest-web/src/main/resources/i18n/application-messages_ja.properties]: OK. I understand the empty message.

**Comment by iwanagah** [comment]
@btkukinom Please backport

**Pull request merged**